docs/contributing/development_environment.md

```diff
@@ -111,11 +111,18 @@ the section:
 [Reset the environment to pre-adoption state](https://openstack-k8s-operators.github.io/data-plane-adoption/contributing/development_environment/#reset-the-environment-to-pre-adoption-state)
 
 ### Create a workload to adopt
+Prep executable for standalone vm
+```
+scp -i ~/install_yamls/out/edpm/ansibleee-ssh-key-id_rsa ~/install_yamls/devsetup/scripts/edpm-deploy-instance.sh root@192.168.122.100:/root/
+```
 
+Creation of workloads next.
 ```
+ssh -i ~/install_yamls/out/edpm/ansibleee-ssh-key-id_rsa root@192.168.122.100
 export OS_CLOUD=standalone
-source ~/install_yamls/devsetup/scripts/edpm-deploy-instance.sh
+source ~/edpm-deploy-instance.sh
 ```
+
 Confirm the image UUID can be seen in Ceph's images pool.
 ```
 sudo cephadm shell -- rbd -p images ls -l
```

Comment:
But the cephadm command here isn't fixed and instead we're editing things above it?

Reply:
Ah yes there is #161 which aims to fix #133. I'm not sure if #114 meant we need to move the edpm-deploy-instance.sh. It may be that those are actually duplicate issues and we have duplicate PRs but neither is editing the intended spot? :)

Reply:
Ah yes #114 was reported at a time when this `edpm-deploy-instance.sh` part didn't exist yet, that's why it's not more specific on what needs to be done. So to my knowledge we only need to fix that `cephadm shell` part as #133 suggests. And none of the open PRs presently address that part. So IIUC we only need a oneliner edit there.